README.md

```diff
@@ -79,6 +80,41 @@ descheduler. See the [helm chart README](https://github.com/kubernetes-sigs/desc
 
 The descheduler helm chart is also listed on the [helm hub](https://hub.helm.sh/charts/descheduler/descheduler-helm-chart).
 
+### Install Using Kustomize
+
+You can use the `kustomization.yaml` file as shown in the following example
```

Comment:
This section is describing how to deploy `descheduler` by third parties rather than how to deploy it from this repository. You should be able to deploy it from scratch simply by running:
```
$ cd kubernetes/[cron]job
$ kustomize build . | kubectl apply -f -
```

If you wanna describe usage through third parties, I'd prefer to create a separate document and mention the document here (as in the helm charts case).

Still, +1 for having just a single kustomize file that can live outside the repo and can be basically run from everywhere.

Reply:
Changed the way descheduler can be deployed more easily.